server/index.js

```diff
@@ -1,58 +1,118 @@
-const express = require('express');
-const path = require('path');
-const bodyParser = require('body-parser');
-const db = require('../database/index.js');
+const newrelic = require('newrelic');
+const cluster = require('cluster');
+// if (cluster.isMaster) {
+//   var cpuCount = require('os').cpus().length;
+//     for (var i = 0; i < cpuCount; i += 1) {
+//         cluster.fork();
+//     }
+// } else {
 
-const app = express();
-
-app.use(bodyParser.json());
-
-app.use(express.static(path.join(__dirname, '/../public')));
-const Port = 3001;
-
-app.get('/messages/:id', (req, res) => {
-  console.log(req.params.id)
-  req.params.id = req.params.id || 1;
-  db.getMessages(req.params.id, (err, data) => {
-    if (err) {
-      res.status(500).send(err);
-    } else {
-      res.status(200);
-      res.send(data);
-    }
+  const express = require('express');
+  const path = require('path');
+  const bodyParser = require('body-parser');
+  const db = require('../database/index.js');
+  const PG = require('../database/indexPG.js');
+  const cors = require('cors');
+  
+  const app = express();
+  app.use(cors());
+  app.use(bodyParser.json());
+  
+  app.use(express.static(path.join(__dirname, '/../public')));
+  const Port = 3001;
+  
+  app.get('/messages/:id', function getting(req, res) {
+    req.params.id = req.params.id || 1;
+    PG.getMessages(req.params.id, (err, data) => {
+      if (err) {
+        res.status(500).send(err);
+      } else {
+        res.status(200);
+        res.send(data);
+      }
+    });
   });
-});
-
-app.post('/messages', (req, res) => {
-  db.createMessage(req.body, (err, result) => {
-    if (err) {
-      res.status(500).send(err);
-      return;
-    }
-    res.status(201).json(result);
+  
+  app.post('/messages', function posting (req, res) {
+    // console.log(req.body);
+    PG.createMessage(req.body, (err, result) => {
+      if (err) {
+        newrelic.noticeError(err);
```

Comment:
I've never seen the ```newrelic.noticeError(err)``` syntax. I would love to chat about any new testing options you've investigated.